content/en/news/releases/1.6.x/announcing-1.6.6/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Announcing Istio 1.6.6
+linktitle: 1.6.6
+subtitle: Patch Release
+description: Istio 1.6.6 patch release.
+publishdate: 2020-07-29
+release: 1.6.6
+aliases:
+    - /news/announcing-1.6.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.5 and Istio 1.6.6.
+
+{{< relnote >}}
+
+## Changes
+
+- **Optimized** `ResolveGatewayName` to greatly reduce the CPU usage and number of memory allocations.
+- **Fixed** `istioctl upgrade` so that it no longer checks remote component versions on dry run. ([Issue 24865](https://github.com/istio/istio/issues/24865))
+- **Fixed** Gateway missing error to produce a short error message even when several clusters with many gateways are present in the configuration.
+- **Fixed** route configuration generation to greatly reduce the load when processing virtual services on gateways. ([Issue 25116](https://github.com/istio/istio/issues/25116))
+- **Fixed** outlier detection to only fire on gateway errors and not depend on success rate.([Issue 25220](https://github.com/istio/istio/issues/25220))
+- **Fixed** demo profile to use port 15021 as the status port. Also performed general cleanup of unused resources. ([Issue #25626](https://github.com/istio/istio/issues/25626))
```

Comment:
nit: cleanup is not user facing really

Reply:
Ok. I listed that because it looks like it's changing the default values in manifests. It looked like they are disabled by default, but if enabled, then the settings would still apply.